.gitignore

```diff
@@ -1,2 +1,3 @@
 waspLineReader.zip
 extension/contentScript.js
+docs/contentScript.js
```

Comment:
Does the addition of this line change the behaviour of git? It would seem that, since that line is already being tracked in git, it can't be ignored using .gitignore without also removing the tracking.

Reply:
I _think_ that will just make it so Git won't track future updates of that file. Considering now it won't work without those event listeners, that's what I want.